source/css/enterprise/db-ui-core.scss

```diff
@@ -1,3 +1,5 @@
+// the
```

Comment:
@annsch are we missing an important comment here ?

Reply:
#Covfefe and removed ;)